app/generic/jsonld/jsonld.py

```diff
@@ -132,6 +115,22 @@ async def verify_jsonld(
 
         # NOTE: Wrong/incomplete aca-py openAPI spec results in wrong/overly-strict model for controller endpoint
         # Hence, override it here
+        # Custom Override api endpoint that is incorrect in aca-py
+        class JsonldApi(Consumer):
```

Comment:
In principal I agree with that. I put it in place for 2 reasons mainly:

1. This is a temporary problem so doing this *should* be superfluous in the near future.
2. Since 1. This makes it more clear why this is done (it is done where is happens ass opposed to somewhere more abstracted/deeper down)

Regardless, happy to move it according to your suggestion if you have a strong preference. I'm fine with either.

Reply:
and forgot to add - please correct me if I'm wrong here: When making an api call the agent_selector creates and yields a new instance of the AcaPyClient (based on auth) every time. So it shouldn't make much of a difference in terms of resource usage?!

Reply:
But maybe I'm missing sth?